playbooks/group_vars/mongodb/public.yml

```diff
@@ -53,3 +53,11 @@ consul_service_config:
       - name: "Connect to MongoDB"
         tcp: "{{ MONGODB_SERVER_DOMAIN }}:27017"
         interval: "10m"
+
+# FILEBEAT ####################################################################
+
+filebeat_prospectors:
+  - fields:
+      type: "mongodb"
+    paths:
+      - "{{ mongodb_systemlog_path }}"
```

Comment:
Is this file rotated by logrotate?  If so, do we need to add an asterisk at the end to also match the rotated files?

Reply:
I don't see anything being rotated in that directory, so I don't think so. Actually, I don't know if I added a comment on this or not, but there's another, much larger file in that directory, `mongodb.log`.

```
1.7G	mongodb.log
8.0K	mongod.log
```

I suppose I should just define a new variable that leads to the log directory, and get `{{ log_dir }}/*.log`.

Reply:
Yeah, this looks rather broken.  I suggest simply going with `"{{ mongodb_systemlog_path | dirname }}/*.log*"` or similar for now, which should cover everything that's relevant.